### DISCUSSION TITLE
LND wont run after Update

### DISCUSSION THREAD

**Discussion by Tanki90** [raspiblitz/raspiblitz#4302] [Q&A] [answered]
Hello,
I had Raspiblitz version 1.8.0 installed, wanted to update to 1.10.0, then the device froze... after 24 hours I hard rebooted the device and started the device with a new SD card image.

Since then I get the message "Unbe able to determine number river" when starting up.

The new installation goes through, but then lnd doesn't start.
you can also see it in the logs.

Then I installed version 1.8.0 again but the error is still there.

Then I updated to the latest lnd version from the menu.

Nothing helped
Can someone help me, I don't know what the problem is.

debug -l
"
Printing debug logs. Be patient, this should take maximum 2 minutes ..
tail: cannot open '/mnt/hdd/lnd/logs/bitcoin/mainnet/lnd.log' for reading: No such file or directory
Redacting ..
# redacting file: /var/cache/raspiblitz/debug.log
https://l.termbin.com/sm2k

**Comment by Tanki90** [comment, marked as answer]
I think this was the Problem!?

78:  Dec 06 22:01:26 raspberrypi lnd[45579]: failed to load config: /home/bitcoin/.lnd/lnd.conf:33: unknown option: routerrpc.apriorihopprob

**Reply by Slumach** [reply]
How'd you fix this?

**Reply by rootzoll** [reply]
Login per SSH > MAINMENU > SYSTEM > LNDCONF "Edit the lnd.conf for mainnet" ... there delete the unknown option and save/restart with OK & RESTART.
